runtime/moonbeam/src/lib.rs

```diff
@@ -1396,6 +1428,7 @@ construct_runtime! {
 		MaintenanceMode: pallet_maintenance_mode::{Pallet, Call, Config<T>, Storage, Event} = 32,
 		Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 33,
 		Migrations: pallet_migrations::{Pallet, Storage, Config<T>, Event<T>} = 34,
+		MultiBlockMigrations: pallet_multiblock_migrations = 121,
```

Comment:
Seem comment in moonbase runtime.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/11cbd029139905cdfe59317757dad36dee1d43e1